messages/origins/messages.go

```diff
@@ -1,77 +1,77 @@
-  OriginsUpdateFlagHmacRegionName       = "HmacRegionName is a field used in the Amazon Web Services (AWS) API to identify the region in which a particular request is being made. It is used to ensure that requests are routed to the correct region and that the correct authentication credentials are used."
-  OriginsUpdateFlagHmacAccessKey        = "HmacAccessKey is a type of authentication key used to access an API or other secure system. It is a combination of a secret key and a cryptographic hash algorithm, such as SHA-256, to generate a unique signature for each request. The signature is then used to verify the authenticity of the request and ensure that it has not been tampered with. HmacAccessKey is often used in combination with other authentication methods, such as OAuth or API keys, to provide an additional layer of security."
-  OriginsUpdateFlagHmacSecretKey        = "HmacSecretKey is a type of cryptographic key used in the HMAC (Hash-based Message Authentication Code) algorithm. It is a secret key that is used to generate a cryptographic hash of a message, which is then used to verify the authenticity and integrity of the message. The key is typically a string of random characters that is known only to the sender and receiver of the message."
-	OriginsUpdateFlagIn                   = "Path to a JSON file containing the attributes that will be updated; you can use - for reading from stdin"
-	OriginsUpdateOutputSuccess            = "Update origin with ID %s\n"
-	OriginsUpdateHelpFlag                 = "Displays more information about the create subcommand"
+	OriginsUpdateShortDescription         = "Updates an Origin"
+	OriginsUpdateLongDescription          = "Updates an Origin based on its ID and given attributes"
+	OriginsUpdateFlagOriginKey            = "The Origin's key unique identifier"
+	OriginsUpdateFlagEdgeApplicationId    = "Unique identifier for an edge application"
+	OriginsUpdateFlagName                 = "The origin's name"
+	OriginsUpdateFlagOriginType           = `Identifies the source of a record. I.e. "single_origin"`
+	OriginsUpdateFlagAddresses            = "Passes a list of addresses linked to the origin"
+	OriginsUpdateFlagOriginProtocolPolicy = "Tells the protocol policy used in the origin"
+	OriginsUpdateFlagHostHeader           = "Specifies the hostname of the server being accessed"
+	OriginsUpdateFlagOriginPath           = "Gives a file path to the origin"
```

Comment:
Origin Path from RTM: Azion can request your content from a directory in your origin. Azion appends Origin Path to the URI when forwarding the request to your origin. Leave it in blank to use only the URI.